dts_portal/dts-portal.js

```diff
@@ -43,8 +44,12 @@ $(document).on('knack-view-render.view_76', function(event, page) {
   hideFieldIfNotRole('kn-input-field_100', 'object_5'); // close date
   hideFieldIfNotRole('kn-input-field_107', 'object_5'); // request type
   hideFieldIfNotRole('kn-input-field_186', 'object_5'); // dataset
+  hideFieldIfNotRole('kn-input-field_294', 'object_5'); // service group
```

Comment:
this might be a little more readable if you gave a name to `object_5`.

```js
var nameOfRole = 'object_5';

hideFieldIfNotRole('kn-input-field_294', nameOfRole); 
...
```

Could also recommend the same idea for fields.
```js
var field = {
  serviceGroup: 'kn-input-field_294',
  dataset: 'kn-input-field_186',
  ...
};

hideFieldIfNotRole(field.dataset, 'object_5'); 
```